package.json

```
@@ -3,74 +3,65 @@
   "version": "3.0.0-beta.1",
   "private": true,
   "description": "Use an Agent to interact with the Internet Computer from your JavaScript program.",
-  "workspaces": {
-    "packages": [
-      "packages/principal",
-      "packages/candid",
-      "packages/agent",
-      "packages/identity",
-      "packages/auth-client",
-      "packages/assets",
-      "packages/identity-secp256k1",
-      "packages/use-auth-client",
-      "e2e/node"
-    ]
-  },
   "type": "module",
   "devDependencies": {
+    "@eslint/eslintrc": "^3.3.1",
+    "@eslint/js": "^9.29.0",
+    "@peculiar/webcrypto": "^1.5.0",
     "@size-limit/esbuild": "^11.2.0",
     "@size-limit/preset-small-lib": "^11.1.6",
     "@tsconfig/node18": "^18.2.4",
     "@types/jest": "^29.5.14",
     "@types/node": "^20.8.6",
+    "@types/text-encoding": "^0.0.40",
     "@typescript-eslint/eslint-plugin": "^8.32.1",
     "@typescript-eslint/parser": "^8.32.1",
+    "dotenv": "^16.5.0",
+    "esbuild": "^0.25.5",
     "eslint": "^9.8.0",
-    "eslint-plugin-cypress": "^3.4.0",
     "eslint-plugin-jsdoc": "^48.11.0",
     "eslint-plugin-prettier": "^5.2.1",
+    "fake-indexeddb": "^6.0.1",
     "husky": "^8.0.3",
     "jest": "^29.7.0",
     "jest-cli": "^29.7.0",
     "jest-environment-jsdom": "29.7.0",
     "jest-environment-node": "^29.7.0",
     "markdown-to-html-cli": "^4.1.0",
-    "os": "^0.1.2",
-    "path": "^0.12.7",
     "prettier": "^3.3.3",
     "prettier-plugin-motoko": "^0.8.4",
     "pretty-quick": "^4.0.0",
     "release-it": "^19.0.2",
     "size-limit": "^11.1.6",
     "size-limit-node-esbuild": "^0.3.0",
+    "text-encoding": "^0.7.0",
     "ts-jest": "^29.3.4",
+    "typedoc": "^0.28.5",
     "typescript": "5.8.3",
-    "url": "^0.11.3",
-    "webcrypto-core": "^1.7.7",
-    "yargs": "^17.7.2",
-    "zx": "^8.1.2"
+    "vitest": "^3.2.4"
   },
+  "packageManager": "pnpm@10.10.0+sha512.d615db246fe70f25dcfea6d8d73dee782ce23e2245e3c4f6f888249fb568149318637dca73c2c5c8ef2a4ca0d5657fb9567188bfab47f566d1ee6ce987815c39",
   "engines": {
     "node": "^20 || ^22",
-    "npm": ">=8"
+    "pnpm": ">=10"
```

Maybe add a notice as usual?
```suggestion
    "pnpm": ">=10",
    "npm": "please use pnpm"
```